index.html

```diff
@@ -11,14 +11,59 @@
     integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
     crossorigin="anonymous"
     referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Forum&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
   <header>
+    <div class="header column-full">
+      <h2>Code Journal</h2>
+    </div>
   </header>
   <main>
+    <div data-view="entry-form">
+      <form id="create-entry">
+        <div class="row">
```

Comment:
There should be a `container` outside of this `row` somewhere. The `main` element is a good candidate to get the `container` class.